ci/ctests/cases/C48_ATM_gfs_fcst_seg0.yaml

```
@@ -0,0 +1,17 @@
+input_files:
```

```suggestion
{% set RUNTESTS = RUNTESTS | getenv %}
input_files:
```
something like this if one needs test specific local info.  For e.g. in the DA jobs, we will need prior dates that are 6 hours behind `TEST_DATE`.